libs/datamodel/core/src/configuration/generator/loader.rs

```diff
@@ -101,6 +101,16 @@ impl GeneratorLoader {
             arguments.push(ast::Argument::new_string("output", &output));
         }
 
+        if !&generator.experimental_features.is_empty() {
+            let features: &Vec<ast::Expression> = &generator
```

Comment:
I think you can omit the `&` here and then you don't need to call `to_owned` in line 111.